spec/spec_helper.rb

```diff
@@ -73,8 +73,8 @@
       if self.class.metadata[:vcr]
         # replace parallel iterations with the serial one, since VCR can't handle parallel request matching correctly
         module Enumerable
-          def peach_with_index(*)
-            each_with_index
+          def peach_with_index(*, &y)
+            each_with_index &y
```

Comment:
Lint/AmbiguousOperator: Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.